perun-utils/rpc-methods-javadoc-generator/parseRpcMethods.pl

```diff
@@ -93,6 +93,17 @@
 $objectExamples{"List&lt;RichGroup&gt;"} = $listPrepend . $objectExamples{"RichGroup"} . $listAppend;
 $objectExamples{"List<RichGroup>"} = $objectExamples{"List&lt;RichGroup&gt;"};
 
+$objectExamples{"Application} = "{ \"id\" : 12 , \"vo\" : ". $objectExamples{"Vo"} . " , \"type\" : \"INITIAL\" , \"fedInfo\" : \"\" , \"state\" : \"NEW\" , \"extSourceName\" : \"PERUNPEOPLE\" , \"extSourceType\" : \"cz.metacentrum.perun.core.impl.ExtSourceSql\" , \"user\" : " . $objectExamples{"User"} . ", \"beanName\" : \"Application\" }";
+$objectExamples{"List&lt;Application&gt;"} = $listPrepend . $objectExamples{"Application"} . $listAppend;
+$objectExamples{"List<Application>"} = $objectExamples{"List&lt;Application&gt;"};
+
+$objectExamples{"ApplicationsPageQuery"} = "{ \"pageSize\" : 3 , \"offset\" : 0 , \"order\" : \"ASCENDING\" , \"sortColumn\" : \"ID\" , \"includeGroupApplications\" : true , \"searchString\" : \"Doe\" , \"states\" : [\"VERIFIED\" , \"NEW\"] , \"dateFrom\" : \"2011-05-17\", \"dateTo\" : \"2011-05-17\", \"memberId\" : 10 , \"groupId\" : 10 }";
+$objectExamples{"List&lt;ApplicationsPageQuery&gt;"} = $listPrepend . $objectExamples{"ApplicationsPageQuery"} . $listAppend;
+$objectExamples{"List<ApplicationsPageQuery>"} = $objectExamples{"List&lt;ApplicationsPageQuery&gt;"};
+
+$objectExamples{"Paginated&lt;Application&gt;"} = "{ \"offset\" : 0 , \"pageSize\" : 3 , \"totalCount\" : 1 , \"data\" : " . $objectExamples{"List&lt;Application&gt;"} . "  }";
+$objectExamples{"Paginated<Application>"} = @objectExamples{"Paginated&lt;Application&gt;"};
```

Comment:
```suggestion
$objectExamples{"Paginated<Application>"} = $objectExamples{"Paginated&lt;Application&gt;"};
```
This looks like a typo, it is only in Paginated objects - can you please fix it in all objects?